practice-one/src/assets/css/footer.css

```diff
@@ -0,0 +1,60 @@
+.footer-section {
+  background-color: var(--black-color);
+  padding-top: 105px;
+  padding-left: 147px;
+  padding-right: 151px;
+  padding-bottom: 91px;
+}
+.footer-container {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+.footer-item-top {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  width: 100%;
+}
+.footer-item-bottom {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  width: 100%;
+}
+
+.item-top {
+  display: flex;
+  flex-direction: column;
+  gap: 24px;
+}
+
+.contact-us {
+  display: flex;
+  flex-direction: row;
+  gap: 54px;
+}
+.follow-us {
+  display: flex;
+  gap: 43px;
+}
+.break {
+  max-width: 133px;
+  max-height: 77px;
+}
+.break-about-us {
+  max-width: 360px;
+  max-height: 48px;
+}
```

Comment:
Should have a break line between block codes

Reply:
![image](https://github.com/HoangLiarPhuongLe/html_css_training/assets/110210274/54b1ca49-b4d4-4ba4-97c2-24407adbc80f)

I fixed it